### PR TITLE
add skim configuration using bunch crossing to the master 

### PR DESCRIPTION
**Pull request by Moanwar**
#### PR description:
Add a special skim configuration for high pile up runs fill number (7358) which we had in 2018

#### PR validation:
The code successfully compiled and tested through CMSDriver locally test. 

@fwyzard @trtomei

#### if this PR is a backport please specify the original PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by prebello** [comment]
@kfjack could you please check? Thank you

**Comment by kfjack** [comment]
Hi @prebello, this PR contains the same BX skim configuration but to master. I have included the same stuff in PR #26476 plus a RelVal. But my latest modification to RelVal does not work somehow. ie. 
- w/o usual RECO steps: it runs fine but the checks cannot be fully proceed due to lack of RECO information(?)
- w/ usual RECO steps: the process failed. Still debugging.

**Comment by prebello** [comment]
@kfjack is it still pending?

**Comment by kfjack** [comment]
Hi @prebello,

It will require a significant work to have the RelVal working, please see the last comments from @kpedro88 in my PR #26476. So I think we have to decide if we should just merge the skim without a corresponding RelVal or not.

**Comment by prebello** [comment]
@kfjack  I see. could you please refresh my mind about the motivation of removing/changing RECO step? is it actually needed? you have been adding skim test relvals without any problem so far.

**Comment by kfjack** [comment]
Hi @prebello, sure! 

- The motivation of removing RECO steps was simply due to this skim is expected to run without RECO (as far as I understood), unlike the regular skims have to run together with RECO. 
- However if I remove the RECO the automatic checks of RelVal cannot give me a full green light.
- I tried to put RECO steps back, it seems that this special data stream (as expected to run on) will generate a lot errors (see PR #26476).

So in summary: adding RelVal w/o RECO -- the automatic checks will not show all green, since lots of stuff cannot be checked); adding RelVal w/ RECO -- the RECO steps itself already contain errors...

**Comment by prebello** [comment]
thank you for the clarification @kfjack 
let me discuss about it within PPD/PdmV and I will be back to you.

**Comment by prebello** [comment]
please test

**Comment by fabiocos** [comment]
@prebello @zhenhu @chayanit @Moanwar what is the situation of this PR?

**Comment by prebello** [comment]
hi @fabiocos there is a conflict that I cannot understand. @kfjack  you have any update on https://github.com/cms-sw/cmssw/pull/26476 ? If not, I guess we should close this PR. Maybe a fresh one can be opened with the proper fixes, if any?

**Comment by kfjack** [comment]
Hi @prebello I do not have a working update. I think we should close this PR. Many thanks!

**Comment by fabiocos** [comment]
@prebello @kfjack @Moanwar so let's close it for the time being
